src/components/changePlanner/CPDetailView.tsx

```diff
@@ -358,157 +392,196 @@ class CPDetailView extends React.Component<
     }
     return (
       <div className={Classes.DARK + " asset-view"}>
+        <Alert
+          cancelButtonText="Cancel"
+          confirmButtonText="Execute"
+          intent={Intent.WARNING}
+          className={Classes.DARK}
+          isOpen={this.state.isAlertOpen}
+          onCancel={() => this.handleExecuteCancel()}
+          onConfirm={() => this.handleExecute()}
+        >
+          <p>Are you sure you want to execute this change plan? You will not be able to generate a work order for this change plan anymore</p>
+        </Alert>
         <Toaster
           autoFocus={false}
           canEscapeKeyClear={true}
           position={Position.TOP}
           ref={this.refHandlers.toaster}
         />
-        <h1>Change Plan</h1>
-        <div className="detail-buttons-wrapper">
-          <div className={"detail-buttons"}>
-            <AnchorButton
-              minimal
-              intent="primary"
-              icon="document-open"
-              text="Generate Work Order"
-            />
-          </div>
-          {this.loading ? <Spinner /> : null}
+        <h1>
+          Change Plan:
+          {this.state.changePlan ? this.state.changePlan.name : null}
+        </h1>
 
-          <ul className="bp3-list-unstyled">
-            {this.state.modifications.length > 0 ? (
-              this.state.modifications.map(
-                (modification: Modification, index: number) => {
-                  return (
-                    <li>
-                      <Callout
-                        icon={null}
-                        intent={
-                          modification.conflicts &&
-                            modification.conflicts.length > 0
-                            ? Intent.DANGER
-                            : Intent.NONE
+        {this.loading ? <Spinner /> : null}
+        <ul className="bp3-list-unstyled">
+          {this.state.changePlan && this.state.changePlan.execution_time ? (
+            <Callout intent="primary">
+              This change plan was executed at:
```

Comment:
there's no space between `at:` and the execution time

Reply:
i swear i have this fixed on my next PR !!